finagle-thrift/src/main/java/com/twitter/finagle/thrift/thrift/Annotation.java

```diff
@@ -34,19 +34,16 @@
   private static final TField TIMESTAMP_FIELD_DESC = new TField("timestamp", TType.I64, (short)1);
   private static final TField VALUE_FIELD_DESC = new TField("value", TType.STRING, (short)2);
   private static final TField HOST_FIELD_DESC = new TField("host", TType.STRUCT, (short)3);
-  private static final TField DURATION_FIELD_DESC = new TField("duration", TType.I32, (short)4);
 
   public long timestamp;
   public String value;
   public Endpoint host;
-  public int duration;
 
   /** The set of fields this struct contains, along with convenience methods for finding and manipulating them. */
   public enum _Fields implements TFieldIdEnum {
     TIMESTAMP((short)1, "timestamp"),
     VALUE((short)2, "value"),
-    HOST((short)3, "host"),
-    DURATION((short)4, "duration");
+    HOST((short)3, "host");
```

Comment:
the top of this file states:
```
/**
 * Autogenerated by Thrift
 *
 * DO NOT EDIT UNLESS YOU ARE SURE THAT YOU KNOW WHAT YOU ARE DOING
 */
```

your changes seem fine to me, so i just want to bring this up.